te_markdown/terminology.md

```diff
@@ -8,7 +8,33 @@ Test-Editor-Web is a lightweight application for specifying, implementing, and e
 
 ## Overview
 
-<!--- TODO: insert (annotated?) overview image and briefly describe the main areas of the user interface -->
+![Overview](/images/te.overview.svg?sanitize=true){:class="img-responsive"}
+
+### The different areas of the Test-Editor-Web
+In the screenshot above you can see the different areas of the Test-Editor-Web which are divided
+through the different yellow areas. The description of the specific area can be found under the number of this area underneath.   
+
+#### 1. The Test-Navigator area
+In this area the user can specify the structure of the different test artifacts.
+He can create, delete, rename, cut, copy and paste test artifacts and refresh the workspace.
+The user can filter the different test-artifacts, so that just the specific test-artifact is visible which was selected.
+
+#### 2. The Editor area
+This area is the place to write different test artifacts. The editor supports the user with a content completion of the different possibilities during typing and saving your test case e.g. through the shortcut ```STRG + S``` . 
+
+#### 3. The Test-Step-Selector area
+The user can find the different testdriver test steps as an overview in the Test-Step-Selektor area, to define test cases.
+
+#### 4. The Test-Execution-Navigator area
+Here you can excute the chosen testcase through clicking the start button on top left corner of this area. After test execution the results of the separate test case steps are shown in this area.
```

Comment:
```suggestion
Here you can excute the chosen test case by clicking the start button on the top left corner of this area. After test execution, the results of the individual test steps are indicated by either green (success) or red (failure) icons.
```
General remark: you're switching between an impersonal style ("The user can find…") and a personal one ("Here you can execute…"). I would suggest to use one ore the other, consistently throughout the document.